pkgs/development/python-modules/pymc3/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, fetchurl
+, buildPythonPackage
+, Theano
+, gfortran
+, openblas
+, numpy
+, scipy
+, matplotlib
+, h5py
+, pandas
+, patsy
+, tqdm
+, joblib
+}:
+
+buildPythonPackage rec {
+  pname = "pymc3";
+  version = "3.2";
+
+  propagatedBuildInputs = [
+    Theano
+    gfortran
+    openblas
```

Comment:
numpy.blas or scipy.blas

Reply:
not needed, your concerns are very valid